docs/spark/spark-ddl.md

```diff
@@ -98,6 +98,15 @@ CREATE TABLE prod.db.sample
 USING iceberg
 AS SELECT ...
 ```
+```sql
```

Comment:
Can you move the paragraph below up here so it's before the sql snippet? Then I think this is good to merge 👍

Reply:
@samredai sure, here is what it looks like now

<img width="888" alt="image" src="https://user-images.githubusercontent.com/10529123/166852014-7a48dfcb-f8f3-4058-9c4f-661d0536a77d.png">